proxy/pom.xml

```diff
@@ -386,7 +386,7 @@
     <dependency>
       <groupId>com.wavefront</groupId>
       <artifactId>java-lib</artifactId>
-      <version>2022-11.1</version>
+      <version>2023-01.31-alpha-3</version>
```

Comment:
Need to wait for Java-lib official release and update java-lib version before merge

Reply:
@joannak-vmware I have added the latest java-lib.